src/sidebar/services/frame-sync.js

```diff
@@ -292,13 +302,14 @@ export class FrameSyncService {
       if (frame) {
         this._store.destroyFrame(frame);
       }
+      // TODO - Remove the guest connection associated with this frame identifier.
```

Comment:
I left this as a TODO to avoid making this PR larger and because this is a pre-existing issue.